lib/rx/client.rb

```diff
@@ -75,6 +75,19 @@ def get_tracking_history_rx(id)
       Common::Collection.new(::Tracking, json.merge(data: tracking_history))
     end
 
+    ##
+    # Post a list of Prescription refills
+    #
+    # @param ids [Array] an array of Rx ids
+    # @return [Faraday::Env]
+    #
+    def post_refill_rxs(ids)
+      if (result = perform(:post, 'prescription/rxrefill', ids, token_headers))
```

Comment:
oh really? I didn't realize that. We may be the only ones using this client so it may be up to us to turn it back on. Do you think this code has to change to accommodate that?

Reply:
No, this is correct. I'm just commenting on something that surprised me because I thought it might be a good thing for you to know too. If you think we should be caching this, then we should create a ticket to make that happen.